pkg/apis/core/v1alpha1/defaults.go

```diff
@@ -325,6 +351,13 @@ func SetDefaults_Shoot(obj *Shoot) {
 	}
 }
 
+func isDockerRuntime(cri *CRI) bool {
+	if cri != nil {
+		return cri.Name == CRINameDocker
+	}
+	return false
```

Comment:
It's also docker when `cri` is `nil`
```suggestion
	return cri == nil || cri.Name == CRINameDocker
```